test/t_cose_compute_validate_mac_test.c

```diff
@@ -135,7 +135,6 @@ int_fast32_t compute_validate_mac_sig_fail_test()
     struct q_useful_buf_c        payload;
     QCBORError                   cbor_error;
     struct t_cose_mac_validate_ctx verify_ctx;
-    size_t                       tamper_offset;
 
```

Comment:
I think I marked the wrong tamper_offset as unused. Didn't realize there were two. 

This one is needed and the one below is unused. 

It compiles OK for OpenSSl, but not for PSA.

Sorry for the confusion.